terraform/groups/ecs-service/locals.tf

```diff
@@ -80,18 +67,17 @@ locals {
     { "name": "MONGODB_URL", "valueFrom": "${local.service_secrets_arn_map.mongodb_url}"},
     {"name": "STAFFWARE_DB_USER", "valueFrom": "${local.service_secrets_arn_map.staffware_db_user}"},
     {"name": "STAFFWARE_DB_PASSWORD", "valueFrom": "${local.service_secrets_arn_map.staffware_db_password}"}, 
-    {"name": "STAFFWARE_DB_CONNECTIONSTRING", "valueFrom": "${local.service_secrets_arn_map.staffware_db_connectionstring}"},       
+    {"name": "STAFFWARE_DB_CONNECTIONSTRING", "valueFrom": "${local.service_secrets_arn_map.staffware_db_connectionstring}"},
+    { "name": "COOKIE_SECRET", "valueFrom": "${local.secrets_arn_map.web-oauth2-cookie-secret}" },
+    { "name": "COOKIE_SERVER", "valueFrom": "${local.service_secrets_arn_map.cookie_server}" }   
   ]
 
   task_environment = [
-    { "name": "LOG_LEVEL", "value": "${var.log_level}" },
-    { "name": "CHS_URL", "value": "${var.chs_url}" },
     { "name": "CDN_HOST", "value": "//${var.cdn_host}" },
```

Comment:
The CDN_HOST in layout.html already has // prepended to the variable. I don't think it's needed here:

https://github.com/companieshouse/transaction-search-tool/blob/main/src/views/layout.html#L7